misk-hibernate/src/main/kotlin/misk/hibernate/Transacter.kt

```diff
@@ -17,12 +17,16 @@ interface Transacter {
   fun isCheckEnabled(check: Check): Boolean
 
   /**
-   * Starts a transaction on the current thread, executes [block], and commits the transaction.
-   * If the block raises an exception the transaction will be rolled back instead of committed.
+   * Starts a transaction, blocks on the current thread until [block] has finished execution, and
+   * commits the transaction. If the block raises an exception the transaction will be rolled back
+   * instead of committed.
    *
    * If retries are permitted (the default), a failed but recoverable transaction will be
    * reattempted after rolling back.
    *
+   * Unless the timeout is disabled, this transaction will be cancelled and rolled back if it
+   * exceeds the duration specified by [misk.jdbc.DataSourceConfig.transaction_timeout].
```

Comment:
Again, this is not true for Vitess. If the Vitess configured timeout is shorter than the datasource timeout then that will be enforced.